redwood-cli/src/main/kotlin/app/cash/redwood/generator/diffProducingGeneration.kt

```diff
@@ -227,10 +227,11 @@ internal fun generateDiffProducingWidget(schema: Schema, widget: Widget, host: S
               }
               is Children -> {
                 addProperty(
-                  PropertySpec.builder(trait.name, NOTHING, OVERRIDE)
+                  PropertySpec.builder(trait.name, RedwoodWidget.WidgetChildren.parameterizedBy(NOTHING))
+                    .addModifiers(OVERRIDE)
                     .getter(
                       FunSpec.getterBuilder()
-                        .addStatement("throw %T()", Stdlib.AssertionError)
+                        .addStatement("return diffProducingWidgetChildren(%LU)", trait.tag)
```

Comment:
Then here is where the diff-producing widgets return an instance of children that produce diffs, and this is where that tag value moved to.